ci/scripts/get-build-type.sh

```diff
@@ -17,7 +17,6 @@ tgt_branch="$1"
 build_reason="$2"
```

Comment:
You can leave the get-build-type.sh script as-is, then we only need to change the CI config when re-enabling this feature.

Reply:
done, thanks!